.cirrus.yml

```diff
@@ -646,6 +736,40 @@ local_system_test_task: &local_system_test_task
     always: *logs_artifacts
 
 
+# Always run subsequent to integration tests.  While parallelism is lost
+# with runtime, debugging system-test failures can be more challenging
+# for some golang developers.  Otherwise the following tasks run across
+# the same matrix as the integration-tests (above).
+local_system_test_aarch64_task: &local_system_test_task_aarch64
+    name: &std_name_fmt_aarch64 "$TEST_FLAVOR $PODBIN_NAME $DISTRO_NV $PRIV_NAME $TEST_ENVIRON"
+    alias: local_system_test_aarch64
+    # Don't create task for tags, or if using [CI:DOCS], [CI:BUILD], multiarch
+    # Docs: ./contrib/cirrus/CIModes.md
+    only_if: *not_tag_build_docs_multiarch
+    depends_on:
+        - build_aarch64
+        - local_integration_test
+    ec2_instance: *standard_build_ec2_aarch64
+    env:
+        <<: *stdenvars_aarch64
+        TEST_FLAVOR: sys
+    clone_script: *get_gosrc_aarch64
+    setup_script: *setup
+    main_script: *main
+    always: &logs_artifacts_aarch64
```

Comment:
Likely we don't need an arm-specific anchor for this section.  Should be good enough to `always: *logs_artifacts`

Reply:
ack